### ISSUE TITLE
Collection#invoke not working properly in 1.2.0

### ISSUE THREAD

**Issue by BluSyn** [jashkenas/backbone#3641]
Calling `invoke` on a collection in previous versions would properly call all models in the collection as specified. This no longer seems to work in 1.2.0. This caused an issue when upgrading one of my applications.

I have reduced this to a sample case on JSFiddle. Switching between Backbone 1.1.2 and 1.2.0 produces different results.

Backbone 1.1.2: https://jsfiddle.net/s3fx4ebL/
Backbone 1.2.0: https://jsfiddle.net/vup2fp4s/

Full Test Code:
```javascript
var Model = Backbone.Model.extend({
    defaults: {
        'a': 1,
        'b': 2
    }
});
var Collection = Backbone.Collection.extend({
    model: Model,
    updateAll: function(data) {
        return this.invoke('set', data);
    }
});

var mylist = new Collection();

var model1 = new Model();
var model2 = new Model();
mylist.add(model1);
mylist.add(model2);

mylist.updateAll({ 'b': 4 });

// All models should show b: 4
console.log(mylist.toJSON());
```

It's possible this issue also affects other Underscore methods within Collections, but I was not able to reproduce that.

**Comment by jridgewell** [comment]
Dupe of #3610, fixed in #3612. We'll be releasing a v1.2.1 when #3603 and #3613 are accepted.

**Comment by BluSyn** [comment]
Thanks! Sorry for the dupe.
